test/mocks/http/mocks.h

```diff
@@ -206,6 +221,8 @@ class MockStreamDecoderFilterCallbacks : public StreamDecoderFilterCallbacks,
   MOCK_METHOD0(continueDecoding, void());
   MOCK_METHOD2(addDecodedData, void(Buffer::Instance& data, bool streaming));
   MOCK_METHOD0(decodingBuffer, const Buffer::Instance*());
+  MOCK_METHOD3(sendLocalReply_, void(Code code, const std::string& body,
```

Comment:
How is this used? Looks like `sendLocalReply` is overriden above, but doesn't call into the mock method.

Reply:
My intention was to call the mock method from the real implementation (which is needed to keep existing tests expecting `encodeHeaders()` calls working), but forgot to do so. I'll remove the unused mock method for now.